libs/ballot-encoder/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@votingworks/ballot-encoder",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Because this is a backward-compatible change that adds functionality, rather than fixing an issue with existing functionality, we should update the "minor" part of the version rather than the "patch" version. We try to follow https://semver.org.
```suggestion
  "version": "5.1.0",
```